docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md

```diff
@@ -14,16 +14,11 @@
 
 双浮点数上限和下限分为别 `-1.79769e+308` 和 `1.79769e+308`。
 
-## 科学计数法
+**科学计数法**
```

Comment:
I think this violates the markdown lint rules. Please change it back to header.
![image](https://user-images.githubusercontent.com/42762957/81790065-6ba5f680-9537-11ea-9625-8efd96b001cf.png)

Reply:
I agree